models/model_predict.ipynb

```diff
@@ -0,0 +1,580 @@
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Maximum Concepts\n",
+    "\n",
+    "Setting the `max_concepts` parameter will customize how many concepts and their corresponding probability scores the predict endpoint will return. \n",
+    "\n",
+    "If not specified, the predict endpoint will return the top 20 concepts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = Model(Model_URL,\n",
+    "      output_config={\n",
+    "          \"max_concepts\": 3\n",
+    "      })\n",
+    "\n",
+    "model_prediction = model.predict_by_url(DOG_IMAGE_URL, input_type=\"image\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Minimum Prediction Value\n",
+    "\n",
+    "This parameter lets you set a minimum probability threshold for the outputs you want to view for the Predict operation.\n",
+    "\n",
+    "For example if you want to see all concepts with a probability score of .95 or higher, this parameter will allow you to accomplish that."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = Model(Model_URL,\n",
+    "        output_config={\n",
+    "            \"min_value\": 0.95\n",
+    "        })\n",
+    "model_prediction = model.predict_by_url(DOG_IMAGE_URL, input_type=\"image\")\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### By Model Version ID\n",
+    "By specifying version_id in your predict call, you can continue to predict on a previous version, for consistent prediction results. Clarifai also updates its pre-built models on a regular basis.\n",
+    "\n",
+    "Below is an example of how you would set a model version ID and receive predictions from Clarifai's `general-image-recognition` model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_version_id = \"aa7f35c01e0642fda5cf400f543e7c40\"\n",
+    "model_url = f\"https://clarifai.com/clarifai/main/models/general-image-recognition/model_version/{model_version_id}\"\n",
+    "\n",
+    "model = Model(model_url)\n",
+    "\n",
+    "model_prediction = model.predict_by_url(DOG_IMAGE_URL, input_type=\"image\")"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "cl-py-utils",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.10"
+  },
+  "orig_nbformat": 4
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
In the last cell , we can add `Clarifai Resources` like in basics and dataset_upload notebooks..